platform/ios/src/MGLMapView.h

```diff
@@ -272,23 +273,24 @@ MGL_EXPORT IB_DESIGNABLE
 @property (nonatomic, readonly) UIButton *attributionButton;
 
 /**
- Support for style classes has been removed. This property always returns an empty array.
+ Show the attribution and telemetry action sheet.
+
+ This method is called when the user taps on `attributionButton`. This method is
+ made available so that it can be assigned to custom attribution button
+ implementations.
  */
+- (IBAction)showAttribution;
+
+// Support for style classes has been removed. This property always returns an empty array.
```

Comment:
Should we mark these methods `@nodoc` to prevent them from counting against our documentation coverage metric?

Reply:
Got this with https://github.com/mapbox/mapbox-gl-native/pull/10085/commits/f8dda965999457dd104a2054a6212abae5820ee4 — jazzy seems fairly particular about requiring the triple-slash, immediately followed by `:nodoc:`.